src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1239,8 +1239,9 @@ internal static Encoding GetEncoding(string path)
             (char) 21, (char) 22, (char) 23, (char) 24, (char) 25, (char) 26, (char) 28, (char) 29, (char) 30,
             (char) 31, (char) 127, (char) 129, (char) 141, (char) 143, (char) 144, (char) 157 };
 
-        internal static readonly UTF8Encoding utf8NoBom =
-            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
+        internal static readonly UTF8Encoding utf8NoBom = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
+        // encoding which is essentially a passthru
+        internal static readonly Encoding iso8859 = Encoding.GetEncoding(28591);
```

Comment:
You should use the overload that takes a string instead:

```c#
GetEncoding("iso-8859-1");
```

Reply:
fixed